SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Push.cs

```diff
@@ -56,6 +61,20 @@ public void InstanceCheckLaunchedFromNotification(string args)
             }
         }
 
+        public void OnUserIdUpdated(object sender, UserIdUpdatedEventArgs e)
+        {
+            lock (Lock)
```

Comment:
```suggestion
            using (_mutex.GetLock(_mutex.State))
```

If @achocron can confirm :)

Reply:
The syntax might actually not be accurate, please modify locally and not accept that suggestion.